### DISCUSSION TITLE
Best way to horizontally scale milvus

### DISCUSSION THREAD

**Discussion by akashmanna** [milvus-io/milvus#4354] [Q&A and General discussion] [answered]
Hi, I am using milvus 10.2 for large-scale feature searching on a set of ~1M feature vectors. 
I am using pymilvus to make queries. However, the number of queries per second is less when the concurrent requests are very high. So I wanted to scale up by creating a cluster using read-only nodes as replicas. Is there a way to create replicas and route requests from pymilvus dynamically to the existing nodes?

**Comment by BossZou** [comment, marked as answer]
You are recommended to use multi read-only nodes above which a load balancer is deployed. Note that whole read-only nodes need to share metadata and file data.

**Reply by chenliyan** [reply]
I'm trying to deploy a multi-read clusters. Found that the readonly node does not sync segments files.If i inserted vec after the nodes started,  the new vec can not be searched.

Do I have to call LoadCollection before search or regularly ? 
on v0.11.0

**Reply by yhmo** [reply]
0.11.0 doesn't support read/write separation. Please wait 0.11.1 which is in testing phase.
